scripts/sanitycheck

```diff
@@ -736,7 +736,7 @@ class DeviceHandler(Handler):
 
         runner = hardware.get('runner', None)
         if runner:
-            board_id = hardware.get("id", None)
+            board_id = hardware.get("runner-id", hardware.get("id", None))
```

Comment:
Maybe it's deliberate, but `None` is `get()`'s default default, so could also do just `hardware.get("id")`.

Reply:
Could, but I wasn't trying to change the coding style, just add a new capability.  If we want to do that, it should be a separate commit and global within the sanitycheck infrastructure.  IMO.